Formula/lacework-cli.rb

```diff
@@ -7,6 +7,11 @@ class LaceworkCli < Formula
   license "Apache-2.0"
   head "https://github.com/lacework/go-sdk.git", branch: "main"
 
+  livecheck do
```

Comment:
```suggestion
  # There can be a notable gap between when a version is tagged and a
  # corresponding release is created, so we check the "latest" release instead
  # of the just the Git tags.
  livecheck do
```

As mentioned before, let's add a comment to explain why we're using `GithubLatest` even though `stable` is a tag. This helps us to keep track of which of these are necessary and why. [I'm currently working through a bunch of `GithubLatest` checks, so this is especially topical.]